src/engine/enginebuffer.cpp

```diff
@@ -705,6 +713,13 @@ void EngineBuffer::slotControlSlip(double v)
     m_slipEnabled = static_cast<int>(v > 0.0);
 }
 
+void EngineBuffer::slotControlSlipCancel(double v)
+{
+    if (load_atomic(m_slipEnabled)) {
```

Comment:
Now m_slipCancelled can be processed and reset in every processSlip() call, independent from m_slipEnabled. 
m_slipEnabled can be set according to m_slipCancelled in the same function afterwards.